src/com/google/javascript/jscomp/gwt/super/com/google/debugging/sourcemap/SourceMapConsumerFactory.java

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
I thought "super" was a gwt term meaning "Instead of compiling from Java, use this JavaScript code in place of this class". So it seems strange for Java files (even stubs like this) to live under a .../gwt/super/ directory. Is this the normal GWT convention?